tests/components/homekit_controller/test_config_flow.py

```diff
@@ -954,6 +1084,4 @@ def test_valid_pairing_codes(pairing_code):
 
     result = await flow.async_step_unignore({"unique_id": "00:00:00:00:00:01"})
     assert result["type"] == "abort"
-    assert flow.context == {
-        "unique_id": "00:00:00:00:00:01",
-    }
+    assert flow.context == {"unique_id": "00:00:00:00:00:01"}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4744987)